### PR TITLE
Added advanced usage example for a notebook interacting with a Cloud …

### PR DESCRIPTION
**Pull request by nhira**
…TPU cluster

## Fixes / Features
- changed README.md to add reference to advanced usage (xpk-notebooks.md)
-

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

File: xpk-notebooks.md
Comment (yejingxin) [inline xpk-notebooks.md:56]: currently lws only support single slice, let us keep the num_slices=1

File: xpk-notebooks.md
Comment (yejingxin) [inline xpk-notebooks.md:197]: let us remove those comments

File: xpk-notebooks.md
Comment (yejingxin) [inline xpk-notebooks.md:252]: also those two debug lines

**Review by yejingxin — COMMENTED** [review]
Looks good overall, added a few comments to clean up debug code.

**Review by Obliviour — APPROVED** [review]
Approving so that after @yejingxin approves, this can be merged.

**Pull request merged**
